examples/build-using-dockerfile/README.md

```diff
@@ -0,0 +1,12 @@
+# `build-using-dockerfile` example
+
+For people familiar with `docker build` command, `build-using-dockerfile` is provided as an example for building Dockerfiles with BuildKit using a syntax similar to `docker build`.
```

Comment:
Wondering (to prevent confusion) if we should mention that current versions of docker already use buildkit, and they only need to set `DOCKER_BUILDKIT=1`